lib/bootstrap/environment.rb

```diff
@@ -15,6 +15,7 @@ module Environment
     BUNDLE_DIR = ::File.join(LOGSTASH_HOME, "vendor", "bundle")
     GEMFILE_PATH = ::File.join(LOGSTASH_HOME, "Gemfile")
     LOCAL_GEM_PATH = ::File.join(LOGSTASH_HOME, 'vendor', 'local_gems')
+    CACHE_PATH = File.join(LogStash::Environment::LOGSTASH_HOME, "vendor", "cache")
```

Comment:
in the previous 4 lines we use `LOGSTASH_HOME` but in this line it's `LogStash::Environment::LOGSTASH_HOME` please stay consistent unless there is a valid reason in which case they should all be updated.